repl/src/main/scala/ammonite/repl/Storage.scala

```diff
@@ -90,6 +92,41 @@ object Storage{
       }
     }
 
+    def classFilesListSave(pkg: String,
+                           wrapper: String,
+                           dataList: List[(String, String)],
+                           tag: String): Unit = {
+      val dir = pkg + "." + wrapper + "0"
```

Comment:
make a file with "0" appended to wrapperName having hashValue and wrapperName of each block